test/cypress/integration/36-pledges.text.js

```diff
@@ -0,0 +1,19 @@
+describe('Pledges', () => {
+  beforeEach(() => {
+    cy.login({ redirect: `/pledges/new` });
+  });
+
+  it('Create the pledge', () => {
+    cy.get('#name', { timeout: 200000 }).clear().type('samcaspus');
```

Comment:
As per https://docs.opencollective.com/help/contributing/development/testing-with-cypress, please use `data-cy` to get elements in the DOM

Reply:
As most components don't seem to have data-cy attribute would it be fine if I add them to the front end?

Reply:
Yes that's the way to go